### PR TITLE
Docs/readout transpilers and functions

### PR DESCRIPTION
**Pull request by fSosaRey**

### CODE REVIEW

File: src/core/transpile.jl [hunk 1]
```diff
@@ -1844,6 +1844,63 @@ end
...
+Quantum Circuit Object:
+   qubit_count: 2 
+   bit_count: 2 
+q[1]:──✲────✲──
+               
+q[2]:──────────
+               
+
+julia> transpiled_circuit = transpile(transpiler, circuit)
+ERROR: AssertionError: Found multiple Readout on qubit: 1
```
Comment (plerouxanyon) [inline src/core/transpile.jl:1900]:
```suggestion
ERROR: AssertionError: Found multiple `Readouts` on qubit: 1
```

File: src/core/quantum_circuit.jl [hunk 2]
```diff
@@ -267,8 +267,10 @@ yield the same output for any input, up to a global phase.
 Circuits with different ordering of gates that apply on different 
 targets can also be equivalent. 
 !!! note 
-    If there are `Readouts` are present in either `QuantumCircuit`, 
-    they are ignored by `compare_circuit`.
+    If there are `Readouts` are present on either `QuantumCircuit`,
```
Comment (plerouxanyon) [inline src/core/quantum_circuit.jl:270]: Just thinking out loud, should we write `Readout`s or `Readouts`?

File: src/core/transpile.jl (hunk 1 above)
Comment (fSosaRey) [reply src/core/transpile.jl]: updated, along with other places we printout errors in the docs

File: src/core/quantum_circuit.jl (hunk 2 above)
Comment (fSosaRey) [reply src/core/quantum_circuit.jl:272]: I find the former looks like a typo

Comment (plerouxanyon) [reply src/core/quantum_circuit.jl:272]: It does look odd, we'll then stick with `Readouts`.

**Review by plerouxanyon — APPROVED** [review]

**Pull request merged**